engine/apps/api/serializers/user.py

```diff
@@ -31,6 +31,26 @@ class UserPermissionSerializer(serializers.Serializer):
     action = serializers.CharField(read_only=True)
 
 
+class GoogleCalendarSettingsSerializer(serializers.Serializer):
+    create_shift_swaps_automatically = serializers.BooleanField()
+    # # TODO: figure out how to get OrganizationFilteredPrimaryKeyRelatedField to work with many=True
+    # specific_oncall_schedules_to_sync =
+    # specific_oncall_schedules_to_sync = serializers.ListField(
+    #     child=OrganizationFilteredPrimaryKeyRelatedField(
+    #         queryset=OnCallSchedule.objects,
+    #         required=False,
+    #         allow_null=True,
+    #     ),
+    #     required=False,
+    #     allow_null=True,
+    # )
+    specific_oncall_schedules_to_sync = serializers.ListField(
+        child=serializers.CharField(),
+        required=False,
+        allow_null=True,
+    )
```

Comment:
I think for now this is okay (especially because we only expose this via the internal API)